docs/source/release/v5.1.0/framework.rst

```diff
@@ -48,6 +48,9 @@ Algorithms
 - Add parameters to :ref:`LoadSampleShape <algm-LoadSampleShape>` to allow the mesh in the input file to be rotated and\or translated
 - Algorithms now lazily load their documentation and function signatures, improving import times from the `simpleapi`.
 - Deprecated the RecordPythonScript algorithm
+- Add parameter 'WorkspaceNames' to :ref:`PlotAsymmetryByLogValue <algm-PlotAsymmetryByLogValue>` which accepts a range of runs to be processed.
```

Comment:
I'm wondering whether framework is the best place to document these changes. I think putting them in the Muon docs would be more appropriate, @AnthonyLim23  thoughts?

Reply:
These are muon specific, the other instruments don't use asymmetry